### ISSUE TITLE
Serverless Developer Experience 

### ISSUE THREAD

**Issue by lucamezzalira** [lnug/speakers#208]
[Abstract of your talk]
During this session I am going to present the capabilities of AWS Toolkit (plugin for VSCode) in conjunction with AWS CDK to enhance the development feedback loop of a serverless workload.

[A bit about you, e.g. current job, twitter handle, etc.]
Principal Serverless Specialist at AWS 
Twitter: @lucamezzalira

- [x ] I confirm that I will abide by the [Code of Conduct](https://lnug.org/code-of-conduct.html)

- [ x] I confirm my talk is 20 mins long in total.

**Comment by Ankcorn** [comment]
Thanks for the talk submission @lucamezzalira . Our next meetup is going to be the 29th of November. Does that date work for you?

**Comment by lucamezzalira** [comment]
Hey @Ankcorn 
sorry I will be outside the country that date, what about December or January?

**Comment by Ankcorn** [comment]
We are planning our next meetup for January 31st. Would that date work for you?

**Comment by lucamezzalira** [comment]
yep it should work thanks!

**Comment by admataz** [comment]
HI @lucamezzalira  - looks like January isn't coming together for LNUG  - hopefully we can get you in for next month?

**Comment by lucamezzalira** [comment]
yep sure, just let me know the date because I've planned already a few working travels between the 5th and 19th Feb

**Comment by Ankcorn** [comment]
Hey @lucamezzalira ,

Sorry for the tremendously long delay. We do have a date planned and would love to have you. 

It's Wednesday the 22nd of May at the Cloudflare offices in waterloo.

Are you still able to give a talk?

**Comment by lucamezzalira** [comment]
Unfortunately I'll be out of UK for work

**Comment by Ankcorn** [comment]
Thanks for the swift response Luca.

If you would like to give this talk at a future meetup let me know and I will keep prompting when dates come up.

**Comment by admataz** [comment]
Hey @lucamezzalira  - I hope this finds you well?  we are resuming our meetup programme after almost a year out.  It's been a helluva disrupted year - and glad to be getting back to normal.  Would you be interested in bringing your talk? Or another? 

We have a date and venue for 24 September - and are planning for October and November too.

**Comment by lucamezzalira** [comment]
thanks for the offer but from Sept to Dec I'm travelling quite a bit this year so I won't be able to deliver the talk. Thanks anyway for the offer

**Comment by admataz** [comment]
Thanks @lucamezzalira  - safe travels!
